variables.tf

```diff
@@ -124,49 +106,18 @@ variable "enable_network_watcher" {
   description = "Flag to control creation of network watcher."
 }
 
-variable "network_security_group_id" {
+variable "enforcement" {
   type        = string
-  default     = null
-  description = "Id of network security group for which flow are to be calculated"
-}
-
-variable "storage_account_id" {
-  type        = string
-  default     = null
-  description = "Id of storage account."
+  default     = "AllowUnencrypted"
```

Comment:
Why it is set to unencrypted?

Reply:
"This subscription does not support \"DropUnencrypted\" as enforcement policy on encrypted vnet. Please register \"Microsoft.Network/AllowDropUnecryptedVnet\" feature on this subscription." Details=[]
│